src/elements/embedded-video/src/index.tsx

```diff
@@ -0,0 +1,53 @@
+/** @jsx jsx */
+
+import * as React from 'react'
+import { jsx } from 'theme-ui'
+
+import Accordion from '../../accordion/src'
```

Comment:
sorry, didn't notice this the first time - this should be `import Accordion from '@uswitch/trustyle.accordion` and the accordion should be in the dependencies of this component